interpreter/core/core.py

```diff
@@ -49,15 +52,21 @@ def __init__(self):
         self._llm = None
 
         # Load config defaults
-        config = get_config()
-        self.__dict__.update(config)
+        self.extend_config(self.config_file)
 
         # Check for update
         if not self.local:
             # This should actually be pushed into the utility
             if check_for_update():
                 display_markdown_message("> **A new version of Open Interpreter is available.**\n>Please run: `pip install --upgrade open-interpreter`\n\n---")
 
+    def extend_config(self, config_path):
+        if self.debug_mode:
+            print(f'Extending configuration from `{config_path}`')
+
+        config = get_config(config_path)
+        self.__dict__.update(config)
```

Comment:
Moving this into a class method allows it to be called programmatically so that developers can load configs when using `interpreter` in Python scripts, as well.

There's an example of this in [`test_interpreter.py`](https://github.com/KillianLucas/open-interpreter/pull/609/files#diff-d54bbc4862f37675b5d0a8512cf7e7ad8c6d6fbc4f8dcdc3ac5b0e205e90d0f1R31)